### PR TITLE
Utm3732op api Get op

### PR DESCRIPTION
**Pull request by issmith1**
UTM-3732 external spec for Operator-API GET Op. 

(in prev PR I had the wrong base branch).

### CODE REVIEW

File: nuss-operator-api/swagger.yaml [hunk 1]
```diff
@@ -916,27 +916,60 @@ paths:
...
         in: "query"
-        description: "fields"
+        description: "Returns Operations with given states. Eg. state=ACCEPTED,ACTIVE"
         required: false
-        type: "string"
-      - name: "q"
+        type: array
+        maxItems: 4
+        minItems: 13
+        items:
```
Comment (issmith1) [inline nuss-operator-api/swagger.yaml]: each string should have a max len I think

File: nuss-operator-api/swagger.yaml [hunk 2]
```diff
@@ -916,27 +916,60 @@ paths:
...
         in: "query"
-        description: "q"
+        description: "Returns Operations with other than the given states. Eg state_not_eq=CLOSED,CANCELLED"
         required: false
-        type: "string"
+        type: array
+        maxItems: 4
+        minItems: 13
+        items:
+          type: "string"
```
Comment (issmith1) [inline nuss-operator-api/swagger.yaml]: same

File: nuss-operator-api/swagger.yaml [hunk 3]
```diff
@@ -2031,8 +2059,8 @@ definitions:
         format: "int32"
         description: "Number of satellites used in calculating the altitude_gps_wgs84_ft."
       enroute_positions_id:
-        type: "integer"
-        format: "int32"
+        type: "string"
+        format: "uuid"
```
Comment (issmith1) [inline nuss-operator-api/swagger.yaml]: for swagger is this not $uuid?

Comment (punamverma) [reply nuss-operator-api/swagger.yaml]: I see it as uuid utmdomain. Could you confirm.

Comment (issmith1) [reply nuss-operator-api/swagger.yaml]: arg pardon its fine

File: nuss-operator-api/swagger.yaml (hunk 2 above)
Comment (punamverma) [reply nuss-operator-api/swagger.yaml]: fixed these

**Pull request merged**